README3.adoc

```diff
@@ -1111,6 +1137,84 @@ class MyApplication : Application {
 
 CAUTION: Objects that are bound with `autoActivitySingleton` will always be injected according to *the last displayed activity*.
 
+==== The service scope
+
+If you want a singleton that lives only during the lifecycle of a specific `Service`, and not any `Context`, you can use the activity scope.
+
+[source, kotlin]
+.Example: using the service scope
+----
+val kodein = Kodein {
+    bind<Logger>() with serviceSingleton { LogManager.getNamedLogger(it.localClassName) } // <1>
+}
+----
+<1> `it` is the service the object is being created for.
+
+As with the context scope, to retrieve objects bound in the service scope, you need to inject a factory which takes the service as a parameter.
+
+[source, kotlin]
+.Example: retrieving a service scoped singleton
+----
+val logger = kodein.with(service).instance()
+----
+
+==== The broadcast receiver scope
+
+If you want a singleton that lives only during the lifecycle of a specific `BroadcastReceiver`, you can use the broadcast receiver scope.
+
+[source, kotlin]
+.Example: using the broadcast receiver scope
+----
+val kodein = Kodein {
+    bind<Logger>() with broadcastReceiverSingleton { LogManager.getNamedLogger(it.localClassName) } // <1>
+}
+----
+<1> `it` is the broadcast receiver the object is being created for.
+
+To retrieve objects bound in the broadcast receiver scope, you need to inject a factory which takes the broadcast receiver as a parameter.
+
+[source, kotlin]
+.Example: retrieving a broadcast receiver scoped singleton
+----
+val logger = kodein.with(broadcastReceiver).instance()
+----
+
+==== The fragment (and support fragment) scope
+
+If you want a singleton that lives only during the lifecycle of a specific `Fragment`, you can use the fragment scope.
+
+[source, kotlin]
+.Example: using the fragment scope
+----
+val kodein = Kodein {
+    bind<Logger>() with fragmentSingleton { LogManager.getNamedLogger(it.localClassName) } // <1>
+}
+----
+<1> `it` is the fragment the object is being created for.
+
```

Comment:
You should add a small line here to speak about the fact that there's also `supportFragmentSingleton`